mainBoard.py

```diff
@@ -0,0 +1,295 @@
+import shipClass
+import score_class
+
+import os
+import sys
+import pygame
+from pygame.locals import *
+import random
+import math
+if not pygame.font:
+    print('Warning, fonts disabled')
+if not pygame.mixer:
+    print('Warning, sound disabled')
+
+
+class BoardInit:
+    def rotShip(self, myangle, shipNum):
+        """rotate the main ship"""
+        self.ships[shipNum].rotatecw(angle=myangle)
+        """changes the ship object in the list of ships"""
```

Comment:
For general commenting (not docstrings), you should use `# comment` to comment your code. `""" comment """` is generally reserved for docstrings and multiline comments.